app/graphql/types/weather_type.rb

```diff
@@ -121,5 +120,7 @@ class Types::WeatherType < Types::WeatherObject
   field "offset", Int
   field "solarcycles", [Types::SolarcycleType]
 
+  field "moonPhase", Float, deprecation_reason: "Moved to Location"
```

Comment:
nit: This isn't a requirement but it would be helpful for the reviewer if this diff happened back at the original line. When reading the change, seeing them tight together is more helpful to the reader. In this case the change is small so it's not that big of a deal. Just something to think about for future commits/PRs.

Reply:
unless there is a good reason as to why this field definition moved from line 18 down to line 123 of course.

Reply:
The field definition was moved out of `WeatherDailyDetailType`, as the API will no longer be providing it within that scope. Refactoring the schema must also move where queries are being made to, therefore moving moonPhase to WeatherType is to additionally simplify such changes.

Reply:
ooooooooh i see now. wow that Diff didn't help me see that at all. Carry on